src/scripts/extension-utils.js

```diff
@@ -15,9 +15,6 @@ const processMessage = (store, type, payload) => {
       store.commit("setExtensionAvailable")
       store.dispatch("getAddressesFromExtension")
       break
-    case "GET_WALLETS_RESPONSE":
```

Comment:
We might still want this to allow the extension to push new accounts. I.e. if the user creates an account while Lunie is open.

Reply:
OK. In that case, it does it make sense to just remove the async call below?

Reply:
Sounds reasonable. What do you think?

Reply:
Less code! 👍  :)